how-to/customize-workspace/client/src/settings.ts

```diff
@@ -17,6 +18,24 @@ async function getConfiguredSettings(): Promise<CustomSettings> {
 	return manifestSettings;
 }
 
+async function getValidHosts(): Promise<string[]> {
+	if (validManifestHosts === undefined) {
+		const manifestHostsPath = window.location.href.replace("platform/provider.html", "manifest-hosts.json");
+		try {
+			logger.info(`Getting valid hosts for initial settings from ${manifestHostsPath}`);
+			const resp = await fetch(manifestHostsPath);
+			const jsonResults: string[] = await resp.json();
+			validManifestHosts = jsonResults;
+			return validManifestHosts;
```

Comment:
Remove this return and let the flow do the return at the end of the method

Reply:
done.